docs/DeveloperGuide.adoc

```diff
@@ -802,30 +804,57 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a person |remove contacts that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |edit the details of a person
```

Comment:
is this still relevant to your product? if so, you need to make it in sync with the rest of the user stories.